lib/ts/ingredients/emaildelivery/services/smtp.ts

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import OverrideableBuilder from "supertokens-js-override";
+
+export interface SMTPServiceConfig {
+    host: string;
+    from: {
+        name: string;
+        email: string;
+    };
+    port: number;
+    secure?: boolean;
+    password: string;
+}
+
+export interface GetContentResult {
+    body: string;
```

Comment:
```suggestion
    body:{text: string} | {html: string} | {text: string, html: string};
```

Reply:
In our default implementation, we need to provide both the text and html versions